wolfcrypt/src/chacha20_poly1305.c

```diff
@@ -248,4 +253,167 @@ static void word32ToLittle64(const word32 inLittle32, byte outLittle64[8])
 }
 
 
+#ifndef NO_CHACHAPOLY_AEAD_IUF
+int wc_ChaCha20Poly1305_Init(ChaChaPoly_Aead* aead,
+    const byte inKey[CHACHA20_POLY1305_AEAD_KEYSIZE],
+    const byte inIV[CHACHA20_POLY1305_AEAD_IV_SIZE],
+    int isEncrypt)
```

Comment:
check values of the isEncrypt input parameter before using it.

Reply:
A check is not required. A non-zero will be encrypt and zero will be decrypt. Thanks to bit-field `isEncrypt:1`.